### PR TITLE
feat!: add context argument to methods

### PR DESCRIPTION
**Pull request by byashimov**

### CODE REVIEW

File: account_teams_test.go [hunk 1]
```diff
@@ -120,8 +121,8 @@ func setupAccountsTeamsTestCase(t *testing.T) (*Client, func(t *testing.T)) {
 	}))
 
 	apiUrl = ts.URL
-
-	c, err := NewUserClient(UserName, UserPassword, "aiven-go-client-test/"+Version())
+	ctx := context.Background()
```
Comment (Serpentiel) [inline account_teams_test.go:124]: is there a way to initialize this globally for each test?

File: version.go [hunk 2]
```diff
@@ -2,5 +2,5 @@ package aiven
 
 // Version returns aiven-go-client version string
 func Version() string {
-	return "v1.2.0"
+	return "v2.0.0"
```
Comment (Serpentiel) [inline version.go:5]: damn, we need to automate this

Comment (Serpentiel) [reply version.go:5]: I've created an issue #307

**Comment by Serpentiel** [comment]
please put `#major` in the commit's body when merging this to get it released as a major release

File: account_teams_test.go (hunk 1 above)
Comment (byashimov) [reply account_teams_test.go]: Tests must be independent, and might have its very own timeout.

**Review by Serpentiel — APPROVED** [review]
lgtm

**Pull request merged**